dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageFactoryImpl.java

```diff
@@ -475,63 +486,52 @@ private void saveLanguageKeys(String fileLangName, Map<String, String> keys, Set
 			if (tempFile.exists())
 				tempFile.delete();
 			if (tempFile.createNewFile()) {
-				fileReader = new FileInputStream(file);
+				fileReader = Files.newInputStream(file.toPath());
 
 				tempFileWriter = new PrintWriter(tempFilePath, "UTF8");
 
 				for (String k : toDeleteKeys) {
 					keys.remove(k);
 				}
-				
-			
+
 				for(Map.Entry<String,String> newKey : keys.entrySet()) {
 					tempFileWriter.println(newKey.getKey() + "=" + newKey.getValue());
 				}
 			} else {
 				Logger.warn(this, "Error creating properties temp file: '" + tempFilePath + "' already exists.");
 			}
-		} catch (FileNotFoundException e) {
-			throw e;
 		} catch (IOException e) {
 			throw e;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add logic to this catch clause or eliminate it and rethrow the exception automatically. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2737)